src/main/java/TodoList.java

```diff
@@ -12,51 +12,49 @@ public int getListnum(){
         return listnum;
     }
 
-    public int addItem(String line){
+    public int addItem(Task task){
         if(listnum < MAXLISTNUM){
-            // todoList[listnum++] = line;
-            tasks[listnum++] = new Task(line);
+            tasks[listnum++] = task;
             System.out.println(SPLITTER);
-            System.out.println("    added: " + line);
+            System.out.println("    " + "Got it. I've added this task:");
+            System.out.println("      " + task);
+            System.out.println("    " + "Now you have " + listnum + " task" + ((listnum>1) ? "s" : "") + " in the list.");
             System.out.println(SPLITTER);
             System.out.println();
             return 1;
         }else{
-            System.out.println("    ERROR: List overflow!");
-            System.out.println();
+            Duke.printError("List overflow");
             return 0;
         }
     }
 
     public int markItem(int num, boolean mark){
-        System.out.println(SPLITTER);
         if(num < listnum && num >= 0){
             // marks[num] = mark;
             tasks[num].mark(mark);
             if(mark){
-                System.out.println("    Nice! I've marked this task as done:");
+            System.out.println(SPLITTER);
+            System.out.println("    Nice! I've marked this task as done:");
                 System.out.print("      [X] ");
             }else{
-                System.out.println("    OK, I've marked this task as not done yet:");
+            System.out.println(SPLITTER);
+            System.out.println("    OK, I've marked this task as not done yet:");
                 System.out.print("      [ ] ");
             }
             System.out.println(tasks[num].getDescription());
             System.out.println(SPLITTER);
             System.out.println();
```

Comment:
So many printing, I think in if-else statement. How about just making additional function for those printing stacks?